blacklist-call/functions/blacklist-call.js

```diff
@@ -0,0 +1,19 @@
+exports.handler = function(context, event, callback) {
+    // List all blocked phone numbers in quotes and E.164 formatting, separated by a comma
+    let blacklist = event.blacklist;  
```

Comment:
Let's add as an alternative here to pass the blacklist as env variable. So basically:

```js
let blacklist = event.blacklist || context.BLACKLIST || ''
```

Reply:
If you are doing that please also add the `BLACKLIST` value to the README as an option similar like here: https://github.com/twilio-labs/function-templates/blob/master/forward-call/README.md#L5-L11

Also please add a .env file with a placeholder like this: https://github.com/twilio-labs/function-templates/blob/master/forward-call/.env